.circleci/config.yml

```diff
@@ -0,0 +1,61 @@
+version: 2
+jobs:
+  setup:
+    docker:
+      - image: node:10.13.0
+    steps:
+      - checkout
+      - run:
+          name: Initial Setup
+          command: |
+            curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+            python get-pip.py
+            curl -o- -L https://yarnpkg.com/install.sh | bash
+            export PATH="$HOME/.yarn/bin:$PATH"
+            python --version
+            python3 --version                     
+            pip --version          
+
+      - run:
+          name: Installing Dependencies
+          command: |
+            yarn install --frozen-lockfile
+            pip install --user -r requirements.txt
+
+      - save_cache:
```

Comment:
Looks like there might be a Yarn cache we want to add here too

Reply:
could you point out where the yarn cache file is stored. I believe we have one yarn cache file for a system which can be found out by running : 
`yarn cache dir`

Reply:
This might help: https://circleci.com/docs/2.0/yarn/

Reply:
thanks

Reply:
the yarn cache would be stored to `.cache` folder and it is already in our cache config.